### ISSUE TITLE
Integrate a decompiler

### ISSUE THREAD

**Issue by mojo2012** [DonJayamanne/javaVSCode#20]
Im working with a closed source framework that is not very well documented (SAP hybris). I need on-demand decompilation of java classes while stepping through the code.

Do you think this is possible? Some of the java decompilers are written in java (like http://www.benf.org/other/cfr/), so they could be integrated in the debug host.

**Comment by faustinoaq** [comment]
@mojo2012 Maybe something like these 👉 [Java decompiler wrapper](https://github.com/rom1504/node-fernflower) or [JD-Core](http://jd.benow.ca/)

**Comment by mojo2012** [comment]
Yes this would be an option I guess. But the hard part is to write the actual on-the-fly-decmpiling-and-still-allowing-to-debug functionality?

**Comment by faustinoaq** [comment]
I think that a decompiler can overload the debuger. For now, I'm decompiling manually :smiley: 

Maybe another extension?

**Comment by mojo2012** [comment]
Well I wouldn't mind another extension, but it should work seamless ;)
